maths/volume.py

```diff
@@ -378,6 +385,7 @@ def vol_conical_frustum(height: float, radius_1: float, radius_2: float) -> floa
         ...
     ValueError: vol_conical_frustum() only accepts non-negative values
     """
+    # Volume - 1/3 * pi * height * (radius_1 squared + radius_2 squared + radius_1 * radius_2)
```

Comment:
Tests are failing on:
* maths/volume.py:388:89: E501 __line too long__ (94 > 88 characters)

Also, please move this line down to be just above the calculation.  Make sure that the order of the values in the comment and in the calculation are the same.